src/intro/hardware.md

```diff
@@ -8,37 +8,32 @@ Let's get familiar with the hardware we'll be working with.
 <img title="F3" src="../assets/f3.jpg">
 </p>
 
-We'll refer to this board as "F3" throughout this book.
-
 What does this board contain?
 
-- A STM32F303VCT6 microcontroller. This microcontroller has
+- A [STM32F303VCT6](https://www.st.com/en/microcontrollers/stm32f303vc.html) microcontroller. This microcontroller has
   - A single-core ARM Cortex-M4F processor with hardware support for single-precision floating point
     operations and a maximum clock frequency of 72 MHz.
 
   - 256 KiB of "Flash" memory. (1 KiB = 10**24** bytes)
 
   - 48 KiB of RAM.
 
-  - many "peripherals": timers, GPIO, I2C, SPI, USART, etc.
-
-  - lots of "pins" that are exposed in the two lateral "headers".
+  - Many integrated peripherals such as timers, I2C, SPI and USART.
 
-  - **IMPORTANT** This microcontroller operates at (around) 3.3V.
+  - General purpose Input Output (GPIO) and other types of pins accessible through the two rows of headers along side the board.
+  
+  - A USB interface accessible through the USB port labeled "USB USER".
 
-- An [accelerometer] and a [magnetometer][] (in a single package).
+- An [accelerometer](https://en.wikipedia.org/wiki/Accelerometer) as part of the [LSM303DLHC](https://www.st.com/en/mems-and-sensors/lsm303dlhc.html) chip.
 
-[accelerometer]: https://en.wikipedia.org/wiki/Accelerometer
-[magnetometer]: https://en.wikipedia.org/wiki/Magnetometer
+- A [magnetometer](https://en.wikipedia.org/wiki/Magnetometer) as part of the [LSM303DLHC](https://www.st.com/en/mems-and-sensors/lsm303dlhc.html) chip.
 
-- A [gyroscope].
+- A [gyroscope](https://en.wikipedia.org/wiki/Gyroscope) as part of the [L3GD20](https://www.pololu.com/file/0J563/L3GD20.pdf) chip.
 
-[gyroscope]: https://en.wikipedia.org/wiki/Gyroscope
+- 8 user LEDs arranged in the shape of a compass.
 
-- 8 user LEDs arranged in the shape of a compass
+- A second microcontroller: a [STM32F103CBT](https://www.st.com/en/microcontrollers/stm32f103cb.html). This microcontroller is actually part of an on-board programmer / debugger and is connected to the USB port named "USB ST-LINK".
```

Comment:
The STM32F103CBT reference is the one that required the fixing...

Reply:
Can you try to edit my patch? is that enabled?